.release-please-manifest.json

```diff
@@ -1,4 +1,5 @@
 {
   "gax": "4.5.0",
-  "tools": "0.4.6"
+  "tools": "0.4.6",
+  "logging-utils": "0.0.0"
```

Comment:
name and version disagree with what's in package.json (`google-logging-utils` and `0.0.2`) - should these match?

Reply:
Hopefully the 0.0.2 is published experimentally! Also - is this name already taken? https://www.npmjs.com/package/logging-utils

Reply:
So the confusion here, I think, is that the folder is `logging-utils` and the package in npm is `google-logging-utils`.

https://www.npmjs.com/package/google-logging-utils

I think in this file, it was supposed to be the folder, but let me know if that's wrong.

0.0.2 is indeed published experimentally (tag `next`).